### PR TITLE
bugfix: CSS not loading when clicking on logo

### LINKED ISSUE #450: bugfix: CSS not loading when clicking on logo
### Description

When running arguflow search locally, navigating to the url in the browser displays the site with the css. However on clicking the Arguflow logo when we navigate to one of the sub routes as shown. This only occurs in dev mode, i.e. when you start `search` with `yarn dev`. 

image

the page gets redirected to the home page without any css applied to it 

Screenshot from 2023-09-19 09-32-52

OS- Ubuntu 22.04

### Target(s)

`search`

### Requirements to close

Issue likely with tailwind css + Astro integration. Clicking on logo should apply the css when the page is redirected. 

- [ ] provide steps required to reproduce the bug
- [ ] add a fix which prevents the bug from ever occurring 

### Community channels

Matrix is preferred. Reach out on discord or Matrix for further assistance. 

- [discord](https://discord.gg/CuJVfgZf54)
- [matrix](https://matrix.to/#/#arguflow-general:matrix.zerodao.gg)

### PR DESCRIPTION
**Pull request by anikdhabal**
fixes #450 
/claim #450

### CODE REVIEW

File: search/src/pages/create.astro [hunk 1]
```diff
@@ -4,9 +4,6 @@ import CreateNewEvidenceForm from "../components/CreateNewEvidenceForm";
 import Footer from "../components/Footer.astro";
 ---
 
-<head>
-  <script src="/tinymce/tinymce.min.js" :inline></script>
-</head>
```
Comment (skeptrunedev) [inline search/src/pages/create.astro:9]: We cannot remove this. It is needed for the create card and edit card forms.

**Review by skeptrunedev — CHANGES_REQUESTED** [review]

File: search/src/pages/create.astro (hunk 1 above)
Comment (anikdhabal) [reply search/src/pages/create.astro:9]: ooh got it

**Comment by skeptrunedev** [comment]
Same thing as the other PRs here. Please make sure to squash and set `bugfix: CSS not loading when clicking on logo` as commit message before re-requesting a review.

**Comment by anikdhabal** [comment]
> Same thing as the other PRs here. Please make sure to squash and set `bugfix: CSS not loading when clicking on logo` as commit message before re-requesting a review.

Sorry, fixed it🙏

**Comment by anikdhabal** [comment]
I found that the style tag is conflicting with the tailwind. Now it is working fine after removing that. @skeptrunedev

**Comment by fastfists** [comment]
Seen, it wouldn't make sense that conflicting styles cause CSS to break, will test shortly

**Comment by skeptrunedev** [comment]
@anikdhabal going to take another look at this shortly. Appreciate the reasoning on the bug only occurring on index and upload.

**Comment by anikdhabal** [comment]
> @anikdhabal going to take another look at this shortly. Appreciate the reasoning on the bug only occurring on index and upload.

Yes, it occurs when we start search with yarn dev.

**Review by skeptrunedev — APPROVED** [review]
Not sure this was causing the issue, but the change is valid.

**Pull request merged**